src/wc.ts

```diff
@@ -0,0 +1,31 @@
+const completionSpec: Fig.Spec = {
+  name: "wc",
+  description: "World, line, character, and byte count",
+  options: [
+    {
+      name: "-c",
+      description: "Output the number of bytes to the standart input",
```

Comment:
typo: there are other occurrences of this error
```suggestion
      description: "Output the number of bytes to the standard input",
```

Reply:
Good catch!